packages/wrangler/src/assets.ts

```diff
@@ -332,6 +338,49 @@ function logAssetUpload(line: string, diffCount: number) {
 	return diffCount++;
 }
 
+/**
+ * Logs a summary of the assets upload status ("Uploaded <count> of <total> assets"),
+ * and the list of uploaded files if in debug log level.
+ */
+function logAssetsUploadStatus(
+	numberFilesToUpload: number,
+	uploadedAssetsCount: number,
+	uploadedAssetFiles: string[]
+) {
+	const level = logger.loggerLevel;
+	const isDebugLogLevel = LOGGER_LEVELS[level] >= LOGGER_LEVELS.debug;
+	let summary = `Uploaded ${uploadedAssetsCount} of ${numberFilesToUpload} assets`;
+	// if debug level we want to list all the files that were uploaded
+	summary += isDebugLogLevel ? ":" : "";
+
+	logger.info(summary);
+
+	if (isDebugLogLevel) {
+		uploadedAssetFiles.forEach((file) => logger.debug(`✨ ${file}`));
+	}
+}
+
+/**
+ * Logs a summary of files read from a given directory ("Read <count>
+ * files from directory <dir>"), and the list of read files if in
+ * debug log level.
+ */
+function logReadFilesFromDirectory(directory: string, assetFiles: string[]) {
+	const level = logger.loggerLevel;
+	const isDebugLogLevel = LOGGER_LEVELS[level] >= LOGGER_LEVELS.debug;
+	let summary = `✨ Read ${assetFiles.length} files from the assets directory ${directory}`;
+	// if debug level we want to list all the files we read from the directory
+	summary += isDebugLogLevel ? ":" : "";
+
+	logger.info(summary);
+
+	if (isDebugLogLevel) {
+		assetFiles.forEach((file) => {
+			logger.debug(`/${file}`);
+		});
+	}
+}
```

Comment:
Why do these both read the log level? I think it'd be simpler if they just unconditionally called `logger.debug()` (I guess we'd lose the `:`, but that doesn't seem worth it to me for coupling to the internal details of how the log levels work